packages/mui-material/src/Dialog/Dialog.js

```diff
@@ -295,6 +295,7 @@ Dialog.propTypes /* remove-proptypes */ = {
   'aria-labelledby': PropTypes.string,
   /**
    * A backdrop component. This prop enables custom backdrop rendering.
+   * @deprecated Use `components.Backdrop` instead.
```

Comment:
Should we update the component itself to use the `components.Backdrop` and the `componentsProps.backdrop` too?

Reply:
Good point!

Reply:
A feedback from a user left with feedback feature at the bottom of the page:

<img width="691" alt="Screenshot 2022-06-30 at 13 12 34" src="https://user-images.githubusercontent.com/3165635/176674238-b2e7fbf2-dea2-4a39-a378-d1ee7e00ed5e.png">

https://mui.com/material-ui/api/dialog/